packages/dashmate/Dockerfile

```diff
@@ -46,19 +48,13 @@ COPY packages/rs-dpp packages/rs-dpp
 COPY packages/wasm-dpp packages/wasm-dpp
 
 # Build WASM DPP
-RUN --mount=type=cache,sharing=private,target=/root/.cache/sccache \
-    --mount=type=cache,sharing=shared,target=${CARGO_HOME}/registry/index \
-    --mount=type=cache,sharing=shared,target=${CARGO_HOME}/registry/cache \
-    --mount=type=cache,sharing=shared,target=${CARGO_HOME}/git/db \
-    export SCCACHE_SERVER_PORT=$((RANDOM+1025)) && \
-    yarn workspace @dashevo/wasm-dpp build && \
```

Comment:
I think we still need this

Reply:
I removed it to try and figure out if it was the cause of step #39 errors